web/modules/custom/dpl_opening_hours/src/Plugin/rest/resource/v1/OpeningHoursResource.php

```diff
@@ -89,7 +89,7 @@ public function get(Request $request): Response {
 
     try {
       $openingHoursInstances = $this->repository->loadMultiple(
-        $typedRequest->getInt('branch_id'),
+        $typedRequest->getInts('branch_id'),
```

Comment:
```suggestion
        [ $typedRequest->getInt('branch_id') ],
```

If you change it to `getInts()` the resource will accept multiple ints even though the specification now only accepts one.

I think it would be more correct to use `getInt()` here as this will fail if the parameter does not contain a single integer.

Then we can always wrap it in an array to comply with the updated `loadMultiple()` signature.